core/meterer/dynamodb_metering_store.go

```diff
@@ -332,3 +332,32 @@ func parsePeriodRecord(bin map[string]types.AttributeValue) (*pb.PeriodRecord, e
 		Usage: uint64(binUsageValue),
 	}, nil
 }
+
+func (s *DynamoDBMeteringStore) GetReservationBinUsage(ctx context.Context, accountID gethcommon.Address, startPeriod uint64, limit int32) ([]*pb.PeriodRecord, error) {
```

Comment:
function looks good. wondering if it is worth refactoring the existing `GetPeriodRecords` to take in a limit, instead of adding a method, since the functionality is largely the same

Reply:
refactored